src/components/FormImageUpload/FormImageUpload.jsx

```diff
@@ -0,0 +1,45 @@
+import * as S from './formImageUpload.styles';
+import plusImg from '../../assets/icons/ic_plus.png';
+
+const FormImageUpload = ({
+  imagePreview,
+  onClick,
+  onChange,
+  onRemove,
+  errorMessage,
+  fileInputRef
+}) => {
+  return (
+    <>
+      <label>상품 이미지</label>
+      <div css={S.imageContainerStyle}>
+        <div css={S.formImageBoxStyle} onClick={onClick}>
```

Comment:
💊 제안
label 태그가 더 적절합니다~
```suggestion
        <label css={S.formImageBoxStyle}> // 클릭시 내부의 input이 선택된것과 동일함.
```